contracts/DomainRegistar.sol

```diff
@@ -4,35 +4,25 @@ pragma solidity ^0.8.24;
 // Uncomment this line to use console.log
 // import "hardhat/console.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-
+import "./DomainUtils.sol";
 
 error AccessDenied(string description);
```

Comment:
line 9 - 11
errors uot of contracts scope
// - 1 point

Reply:
please clarify here. Do you mean a lib function should not revert byself and only communicate to the contract if call passed/failed?

Reply:
no 
it should have a place inside in contract scope 

contract DomainRegistar is Initializable {
    error ... 
}

Reply:
moved error defs to a contract file, please check